examples/va/hci/values.yaml

```diff
@@ -0,0 +1,21 @@
+# local-config: referenced, but not emitted by kustomize
+apiVersion: v1
+kind: ConfigMap
+metadata:
```

Comment:
As discussed, would it be possible to add a new entry in the values.yml `metadata` dict? Something like:
```YAML
metadata:
  name: edpm-values-post-ceph
  label: dataplane (or controlplane, or services [provided we can kustomize that one])
```
Doing so would make automation slightly less complicated, allowing to detect the "type|kind|whatever" we're trying to deploy.

Reply:
Yes, we can add arbitrary labels using the standard k8s format.  It would look something like so:

```
metadata:
  name: edpm-values-post-ceph
  labels:
    dataplane: true
```

Reply:
Sounds good! Label names such as `dataplane` and `controlplane`(and, maybe, `service`?) sounds like a good plan - and then I can `metadata.labels.get('dataplane', False)` and I get my "type".

Reply:
I'm just wondering which particular labels would be appropriate for which `*values.yaml` files.  In the case of the control plane, I think that is pretty obvious.  But this `edpm-values-post-ceph` is a little trickier...maybe.  It has stuff for the `OpenStackDataPlaneNodeSet` but also for a `Secret` and ` ConfigMap` -- but they are all indeed related to the data plane.  I guess it depends on what exactly the labels mean to you.

Could you let me know what labels you would like to see on the various `*values.yaml`s in the PR?  I've listed them below and tried to describe what they customize.

- `control-plane/nncp/values.yaml` (`NetConfig`, `NetworkAttachmentDefinition`, `NNCP`, `L2Advertisement` `IPAddressPool` and `OpenStackControlPlane` customizations for initial control plane)
- `edpm-pre-ceph/values.yaml` (`OpenStackDataPlaneNodeSet` and a `Secret` customizations for pre-Ceph, i.e. initial dataplane)
- `service-values.yaml` (`OpenStackControlPlane` customizations for post-Ceph)
- `values.yaml` (`OpenStackDataPlaneNodeSet`, `Secret` and `ConfigMap` customizations for post-Ceph)